api/src/pcapi/routes/backoffice/templates/operations/details.html

```diff
@@ -239,50 +244,75 @@ <h2 class="flex-fill">{{ self.title() }}</h2>
     </a>
   </div>
   <div>
-    {% call description_detail_vertical("Date de l'opération") %}
-      {{ special_event.eventDate |format_date }}
-      {% if has_permission("MANAGE_OFFERS") %}
-        <button type="button"
-                class="btn p-0"
-                data-bs-toggle="tooltip"
-                data-bs-placement="top"
-                data-bs-title="Modifier la date de l'évènement">
-          <span data-bs-toggle="modal"
-                data-bs-target="#update-event-date-modal">
-            <i class="bi bi-pencil-square"></i>
-          </span>
-        </button>
-        {{ build_lazy_modal(url_for("backoffice_web.operations.update_date_event", special_event_id=special_event.id) , "update-event-date-modal") }}
-      {% endif %}
-    {% endcall %}
-    {% call description_detail_vertical("Date de clôture des candidatures") %}
-      {{ special_event.endImportDate |format_date }}
-      {% if has_permission("MANAGE_OFFERS") %}
-        <button type="button"
-                class="btn p-0"
-                data-bs-toggle="tooltip"
-                data-bs-placement="top"
-                data-bs-title="Modifier la date de l'évènement">
-          <span data-bs-toggle="modal"
-                data-bs-target="#update-end-import-date-modal">
-            <i class="bi bi-pencil-square"></i>
-          </span>
-        </button>
-        {{ build_lazy_modal(url_for("backoffice_web.operations.update_end_import_date", special_event_id=special_event.id) , "update-end-import-date-modal") }}
-      {% endif %}
-    {% endcall %}
-    {% call description_detail_vertical("Partenaire culturel") %}
-      {% if special_event.venueId %}
-        <a class="link-primary"
-           data-bs-toggle="tooltip"
-           data-bs-placement="top"
-           data-bs-title="Voir le partenaire culturel"
-           target="_top"
-           href="{{ url_for('backoffice_web.venue.get', venue_id=special_event.venueId) }}">{{ special_event.venue.common_name }}</a>
-      {% else %}
-        <br />
-      {% endif %}
-    {% endcall %}
+    <dl>
+      <dt class="hstack gap-1">
+        <span>Date de l'opération</span>
+        {% if can_manage_event %}
+          <button type="button"
+                  class="btn btn-outline-primary-subtle-bg border-0 px-2 py-1"
+                  data-bs-toggle="tooltip"
+                  data-bs-placement="top"
+                  data-bs-title="Modifier la date de l'évènement">
+            <span data-bs-toggle="modal"
+                  data-bs-target="#update-event-date-modal">
+              <i class="bi bi-pencil-square"></i>
+            </span>
+          </button>
+        {% endif %}
+      </dt>
+      <dd>
+        {{ special_event.eventDate |format_date }}
+      </dd>
+    </dl>
+    <dl>
+      <dt class="hstack gap-1">
+        <span>Date de clôture des candidatures</span>
+        {% if has_permission("MANAGE_OFFERS") %}
```

Comment:
C'est toujours la mauvaise permission :
```suggestion
        {% if can_manage_event %}
```